Change question issue template

jmfrenzel opened the pull request:
Fixes #12 ; refs #11

labels: '"Question Issue"'


@FDM-UARuhr/Reviewers

jmfrenzel committed:
initial version of a issue template for change request: questions

jmfrenzel committed:
formating

jmfrenzel commented:
would add notes to the README.md files that this is a local and modified version of github.com/rdmorganiser/rdmo-catalog

jmfrenzel committed:
jont issue template for new/change rdmo question

Bassbiest commented:
This PR is already outdated. The proposed changes here are also implemented and improved in PR #6

Commit:
Merge remote-tracking branch 'rdmo-catalog-uaruhr/change_question_issue_template' into change_question_issue_template

jmfrenzel commented:
from @Bassbiest 

##### I suggest keeping the default names/values as short as possible, more specific and consistent. I added a few suggestions separated by "/". Furthermore, a lot of the information that should be given here depends on how the data is tracked.

---
name: Question Issue/Catalog Issue
about: Create a report to help us improve a question in RDMO catlog UA Ruhr 
title: 'Question Issue: (path/URI/internal ID)'
labels: '"Question Issue"'

** From the following list delete everything that is NOT content of your proposal. Once a subject of change is approved it will be marked accordingly in the list. **
- [ ] uri
- [ ] key
- [ ] path
- [ ] questionset
  - [ ] position in set
- [ ] question text (en and ger)
- [ ] help text (en and ger)
- [ ] answer
  - [ ] widget type
  - [ ] value type
  - [ ] max/min, step and unit
- [ ] optionset
- [ ] condition
- [ ] attribute

** Explain the issue. **


** Propose what should be done about it one setting at a time**
### question text

### key


---
##### In issue #11 I added a code block because we currently don't have any means to display our data of the question. Another option, instead of a code block, could be a simple reference to [something like this](https://bassbiest.github.io/rdmo-catalog-uaruhr/questions/uar01.html). Again, this strongly depends on our representation of the data.

---

jmfrenzel commented:
name: New/Edit Question
about: Create a report to help us improve a question in RDMO catalog UA Ruhr. Provide suggestions for a new RDMO question or for updating an existing one.
title: 'New/Edit Question: TYPE PATH URI HERE
labels: '"Question Issue"'

jmfrenzel committed:
Update change_question.md

Formatting

jmfrenzel committed:
Update change_question.md

Bassbiest committed:
Update change_question.md
